### DISCUSSION TITLE
Setting polynomial functions for inlet velocity profile.

### DISCUSSION THREAD

**Discussion by akshaymani94** [nextfoam/baram#124] [Q&A]
Is there a way in Baram to set the inlet velocity or temperature profile as a polynomial expression at the boundaries?

**Comment by kdg5424** [comment]
Hello @akshaymani94. This is Donggyu.

Yes there is a way to set the profile as polynmoial but you should do it with some tricks.

For the temperature profile, you can set the profile as a polynomial expression for the temporal distribution.
![image](https://github.com/user-attachments/assets/80130970-d36f-4474-b404-641f6b2f7ee6)
![image](https://github.com/user-attachments/assets/0eb54a09-4201-42c2-b7d5-690da1380c3d)

But for the velocity and temperature (for the spatial profile) in current status, you should make CSV file and import it.
I'm not sure which one you want (spatial profile or temporal profile) but you can configure the polynomial profile as you want.

Best regards

Donggyu Kim
